### PR TITLE
Rename chaos-server to chaos-dashboard

### PR DESCRIPTION
**Pull request by cwen0**
### What problem does this PR solve?  

* rename chaos-server to chaos-dashboard 
* install chaos-dashboard with `install.sh` 

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: helm/chaos-mesh/README.md [hunk 1]
```diff
@@ -54,8 +54,17 @@ The following tables list the configurable parameters of the Chaos Mesh chart an
...
+| `dashboard.persistentVolume.existingClaim` | Use the existing PVC for persisting chaos event| `` |
+| `dashboard.persistentVolume.size` | Chaos Dashboard data Persistent Volume size | `8Gi` |
+| `dashboard.persistentVolume.storageClassName` | Chaos Dashboard data Persistent Volume Storage Class | `standard` |
+| `dashboard.persistentVolume.mountPath` | Chaos Dashboard data Persistent Volume mount root path | `/data` |
+| `dashboard.persistentVolume.subPath` | Subdirectory of  Chaos Dashboard data Persistent Volume to mount | `` |
+| `dashboard.persistentVolume.env` | The keys within the `env` map are mounted as environment variables on the Chaos Dashboard pod | `` |
+| `dashboard.persistentVolume.env.LISTEN_HOST` | | `0.0.0.0` |
+| `dashboard.persistentVolume.env.LISTEN_PORT` | | `2333` |
+| `dashboard.persistentVolume.env.DATABASE_DRIVER`| The db drive used for Chaos Dashboard, support db: sqlite3, mysql| `sqlite3` |
+| `dashboard.persistentVolume.env.DATABASE_DATASOURCE`| The db dsn used for Chaos Dashboard | `/data/core.sqlite` |
```
Comment (Gallardot) [inline helm/chaos-mesh/README.md:67]: fix typo

replace `persistentVolume.env` to `env`

File: Dockerfile [hunk 2]
```diff
@@ -22,4 +22,5 @@ ARG SWAGGER
 
 COPY . /src
 WORKDIR /src
+RUN git checkout .
```
Comment (yeya24) [inline Dockerfile:25]: What is this for?

Comment (cwen0) [reply Dockerfile:25]: we will check the repo status when to build.  such as: `v1.0.0-alpha.1.18+d9be6353138498-dirty`, if the repo was changed, we will add a `dirty` field to the message. So we add `git checkout .` to make sure the repo is clean.

**Pull request merged**